src/api/middleware/headers/security/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response, NextFunction } from 'express';
+
+/**
+ * Global middleware, ensures myriads of imperative security headers.
+ * - `HSTS` - 1 Year
+ * - `X-Frame-Options` - Clickjacking
+ * - `X-Content-Type-Options` - Content-Type headers should be followed
+ * @param {Express.Request} req Request object
+ * @param {Express.Response} res Response object
+ * @returns {Function} next()
+ */
+const security = (req: Request, res: Response, next: NextFunction) => {
+  res.setHeader('Strict-Transport-Security', 'max-age=15552000; includeSubDomains; preload');
```

Comment:
Should we also add remaining security headers from `ui`?

Reply:
Good shout, have added all the same headers as the UI. Now both are aligned 👍